<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Source Themes Academic 4.5.0"><link rel=stylesheet href=https://louisfaure.github.io/css/shield.css><meta name=author content="Louis Faure"><meta name=description content="PhD candidate"><link rel=alternate hreflang=en-us href=https://louisfaure.github.io/><meta name=theme-color content="#0A1922"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.min.css integrity="sha256-+N4/V/SbAFiW1MPBCXnfnP9QSN3+Keu+NlB+0ev/YKQ=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.10/styles/github.min.css crossorigin=anonymous title=hl-light><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.10/styles/dracula.min.css crossorigin=anonymous title=hl-dark disabled><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.5.1/leaflet.css integrity="sha256-SHMGCYmST46SoyGgo4YR/9AlK1vf3ff84Aq9yK4hdqM=" crossorigin=anonymous><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Montserrat:400,700%7CRoboto:400,400italic,700%7CRoboto+Mono&display=swap"><link rel=stylesheet href=/css/academic.css><script async src="https://www.googletagmanager.com/gtag/js?id=UA-175353387-1"></script><script>window.dataLayer=window.dataLayer||[];function gtag(){dataLayer.push(arguments);}
function trackOutboundLink(url){gtag('event','click',{'event_category':'outbound','event_label':url,'transport_type':'beacon','event_callback':function(){document.location=url;}});console.debug("Outbound link clicked: "+url);}
function onClickCallback(event){if((event.target.tagName!=='A')||(event.target.host===window.location.host)){return;}
trackOutboundLink(event.target);}
gtag('js',new Date());gtag('config','UA-175353387-1',{});document.addEventListener('click',onClickCallback,false);</script><link rel=alternate href=/index.xml type=application/rss+xml title="Louis Faure"><link rel=manifest href=/index.webmanifest><link rel=icon type=image/png href=/img/icon-32.png><link rel=apple-touch-icon type=image/png href=/img/icon-192.png><link rel=canonical href=https://louisfaure.github.io/><meta property="twitter:card" content="summary"><meta property="og:site_name" content="Louis Faure"><meta property="og:url" content="https://louisfaure.github.io/"><meta property="og:title" content="Louis Faure"><meta property="og:description" content="PhD candidate"><meta property="og:image" content="https://louisfaure.github.io/img/icon-192.png"><meta property="twitter:image" content="https://louisfaure.github.io/img/icon-192.png"><meta property="og:locale" content="en-us"><meta property="og:updated_time" content="2022-07-10T00:00:00&#43;00:00"><script type=application/ld+json>{"@context":"https://schema.org","@type":"WebSite","potentialAction":{"@type":"SearchAction","target":"https://louisfaure.github.io/?q={search_term_string}","query-input":"required name=search_term_string"},"url":"https://louisfaure.github.io/"}</script><title>Louis Faure</title></head><body id=top data-spy=scroll data-offset=70 data-target=#navbar-main><aside class=search-results id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=#><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><nav class="navbar navbar-light fixed-top navbar-expand-lg py-0 compensate-for-scrollbar" id=navbar-main><div class=container><a class=navbar-brand href=/>Louis Faure</a>
<button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar aria-controls=navbar aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="collapse navbar-collapse" id=navbar><ul class="navbar-nav mr-auto"><li class=nav-item><a class=nav-link href=/#about data-target=#about><span>About</span></a></li><li class=nav-item><a class=nav-link href=/#skills data-target=#skills><span>Skills</span></a></li><li class=nav-item><a class=nav-link href=/#talk data-target=#talk><span>Talks/Poster</span></a></li><li class=nav-item><a class=nav-link href=/#featured data-target=#featured><span>Publications</span></a></li><li class=nav-item><a class=nav-link href=/#experience data-target=#experience><span>Experience</span></a></li><li class=nav-item><a class=nav-link href=/#contact data-target=#contact><span>Contact</span></a></li></ul><ul class="navbar-nav ml-auto"><li class=nav-item><a class="nav-link js-search" href=#><i class="fas fa-search" aria-hidden=true></i></a></li></ul></div></div></nav><span class="js-widget-page d-none"></span><section id=hero class="home-section wg-hero dark" style=background-color:#02224b;background-image:linear-gradient(#02224B,#02224B)><div class=container><header id=custom-header><div id=header_viz></div></header><style>.child{box-sizing:border-box;text-align:center;color:#fff}html,body{width:100%;height:100%;margin:0;padding:0;background-color:#02224b}figure{margin:0;bottom:0;position:relative}figure h1{position:absolute;top:20px;left:20px;border-right:3px solid #fff;padding:5px 10px 5px 5px}video{min-width:1920px;max-width:100%;min-height:1080px;position:fixed;left:50%;top:100%;transform:translate(-50%,-100%);-moz-transform:translate(-50%,-100%);-webkit-transform:translate(-50%,-100%)}.stretch{height:calc((60vw - 400px) * 0.5625);background:0 0;min-height:0%;min-width:100%;background-size:cover;margin-top:0;display:table}.stretch h1{padding:20px}.hero-wrapper{position:fixed;bottom:0;right:0;height:auto;width:auto;min-height:100%;min-width:100%;max-height:100%;z-index:-9999}.child{display:table-cell;vertical-align:middle}</style><script src=//ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js></script><script src=https://vjs.zencdn.net/7.8.4/video.js></script><div class=hero-wrapper><div class=hero><figure><video class=patient-video poster=img/headers/1stframe.jpg loop=loop muted autoplay style=max-height:100%!important>
<source data-src=img/headers/media.mp4 src type=video/mp4></video></figure></div></div><div class=stretch><div class=child><h1 style="text-shadow:5px 5px 4px #000!important;font-family:lato;font-size:calc(12px + 3vw)">Unraveling transcriptional flow from single cell transcriptomic data...</h1></div></div><script>(function(a){(jQuery.browser=jQuery.browser||{}).mobile=/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))})(navigator.userAgent||navigator.vendor||window.opera);if($.browser.mobile)
{console.log("is mobile")}
else
{console.log("is desktop")
var sources=document.querySelectorAll('video.patient-video source');var video=document.querySelector('video.patient-video');for(var i=0;i<sources.length;i++){sources[i].setAttribute('src',sources[i].getAttribute('data-src'));}
video.load();video.muted="muted";}</script></div></section><section id=about class="home-section wg-about" style="padding:20px 0"><div class=container><div class=row><div class="col-12 col-lg-4"><div id=profile><img class=portrait src=/authors/admin/avatar_hu45bfa72f15e5c666049c45e54da865d3_79677_250x250_fill_q90_lanczos_center.jpg alt=Avatar><div class=portrait-title><h2>Louis Faure</h2><h3>PhD candidate</h3><h3><span>Medical University of Vienna</span></h3></div><ul class=network-icon aria-hidden=true><li><a href=mailto:louis.faure@meduniwien.ac.at><i class="fas fa-envelope big-icon"></i></a></li><li><a href=https://orcid.org/0000-0003-4621-586X target=_blank rel=noopener><i class="fab fa-orcid big-icon"></i></a></li><li><a href=https://www.researchgate.net/profile/Louis-Faure target=_blank rel=noopener><i class="fab fa-researchgate big-icon"></i></a></li><li><a href="https://scholar.google.com/citations?user=Tc8i56oAAAAJ" target=_blank rel=noopener><i class="ai ai-google-scholar big-icon"></i></a></li><li><a href=https://github.com/louisfaure target=_blank rel=noopener><i class="fab fa-github big-icon"></i></a></li><li><a href=https://soundcloud.com/louisfaure target=_blank rel=noopener><i class="fab fa-soundcloud big-icon"></i></a></li></ul></div></div><div class="col-12 col-lg-8"><h2>About me</h2><p>My PhD aims to to expand our views on the evolution of neural crest properties. More precisely by unravelling the cell fate decisions of neural crests cells during mouse development and by exploring the heterogeneity of glial cells during development and adulthood in mouse.</p><p>My current work is mainly based on the usage of bioinformatic tools such as single-cell transcriptomics data analysis, clustering, pseudotime ordering, combined with biological experiments for validation.</p><div class=row><div class=col-md-5><h3>Interests</h3><ul class=ul-interests><li>Computational Biology</li><li>Developmental Biology</li><li>Evolution</li><li>Exo-biology</li></ul></div><div class=col-md-7><h3>Education</h3><ul class="ul-edu fa-ul"><li><i class="fa-li fas fa-graduation-cap"></i><div class=description><p class=course>PhD candidate in Neuroscience, 2018</p><p class=institution>Medical University of Vienna</p></div></li><li><i class="fa-li fas fa-graduation-cap"></i><div class=description><p class=course>Diploma Institute for Technology Innovation, 2017</p><p class=institution>PSL University, Paris</p></div></li><li><i class="fa-li fas fa-graduation-cap"></i><div class=description><p class=course>Msc in Interdisciplinary Approaches in Life Science, 2016</p><p class=institution>Center for Research and Interdisciplinarity, Paris</p></div></li><li><i class="fa-li fas fa-graduation-cap"></i><div class=description><p class=course>BSc in Life Science, 2013</p><p class=institution>Center for Research and Interdisciplinarity, Paris</p></div></li></ul></div></div></div></div></div></section><section id=skills class="home-section wg-featurette" style="padding:20px 0"><div class=container><div class="row featurette"><div class="col-md-12 section-heading"><h1>Skills</h1></div><div class="col-12 col-sm-4"><div class=featurette-icon><img src=/images/icon-pack/python.svg alt=python class="svg-icon svg-baseline"></div><h3>Python</h3></div><div class="col-12 col-sm-4"><div class=featurette-icon><img src=/images/icon-pack/R.svg alt=R class="svg-icon svg-baseline"></div><h3>R</h3></div><div class="col-12 col-sm-4"><div class=featurette-icon><img src=/images/icon-pack/Tux.svg alt=Tux class="svg-icon svg-baseline"></div><h3>Linux</h3></div><div class="col-12 col-sm-4"><div class=featurette-icon><img src=/images/icon-pack/server.svg alt=server class="svg-icon svg-baseline"></div><h3>Server managment</h3></div><div class="col-12 col-sm-4"><div class=featurette-icon><img src=/images/icon-pack/gpu3.svg alt=gpu3 class="svg-icon svg-baseline"></div><h3>GPU data analysis</h3></div><div class="col-12 col-sm-4"><div class=featurette-icon><img src=/images/icon-pack/Fiji.svg alt=Fiji class="svg-icon svg-baseline"></div><h3>ImageJ</h3></div><div class="col-12 col-sm-4"><div class=featurette-icon><img src=/images/icon-pack/insitu.svg alt=insitu class="svg-icon svg-baseline"></div><h3>FISH techniques</h3></div><div class="col-12 col-sm-4"><div class=featurette-icon><img src=/images/icon-pack/Cloning.svg alt=Cloning class="svg-icon svg-baseline"></div><h3>Cloning</h3></div><div class="col-12 col-sm-4"><div class=featurette-icon><img src=/images/icon-pack/ableton.svg alt=ableton class="svg-icon svg-baseline"></div><h3>Music production</h3></div></div></div></section><section id=talk class="home-section wg-pages" style="padding:20px 0 50px"><div class=container><div class=row><div class="col-12 col-lg-4 section-heading"><h1>Talks/Posters</h1></div><div class="col-12 col-lg-8"><div class=view-list-item><i class="far fa-calendar-alt pub-icon" aria-hidden=true></i><a href=/talk/fens/>Poster for FENS: Schwann cell precursors represent a neural crest-like state with biased multipotency</a><div class=article-metadata><span>Jul 9, 2022 &mdash; Jul 13, 2022</span>
<span class=middot-divider></span><span>Paris, France</span></div></div><div class=view-list-item><i class="far fa-calendar-alt pub-icon" aria-hidden=true></i><a href=/talk/embl/>EMBL-EBI lecture: Introduction to trajectory inference techniques</a><div class=article-metadata><span>Mar 4, 2022 10:00 AM &mdash; 3:00 PM</span>
<span class=middot-divider></span><span>Online</span></div></div><div class=view-list-item><i class="far fa-calendar-alt pub-icon" aria-hidden=true></i><a href=/talk/bioinfokarolinska/>Seminar: Transcriptional landscape of fate choices in the sensory lineages</a><div class=article-metadata><span>Jun 16, 2020</span>
<span class=middot-divider></span><span>Online</span></div></div></div></div></div></section><section id=featured class="home-section wg-featured" style="padding:20px 0"><div class=container><div class=row><div class="col-12 col-lg-4 section-heading"><h1>1st Author Publications</h1></div><div class="col-12 col-lg-8"><div class=card-simple><div class=article-metadata><div><span class=font-weight-bold><a href=/authors/admin/>L. Faure</a></span>, <span>R. Soldatov</span>, <span>P. V. Kharchenko</span>, <span>I. Adameyko</span></div><span class=article-date>July 2022</span>
<span class=middot-divider></span><span class=pub-publication><em>bioRxiv</em></span></div><a href=/publication/faure2022/><img src=/publication/faure2022/featured_hu6a7e5c640ce1396b72b3025c016be542_70624_918x517_fill_q90_lanczos_smart1.jpg class=article-banner alt></a><h3 class="article-title mb-1 mt-3"><a href=/publication/faure2022/>scFates: a scalable python package for advanced pseudotime and bifurcation analysis from single cell data</a></h3><div class=article-style><p>scFates provides an extensive toolset for analysis of dynamic trajectories comprising tree learning, feature association testing, branch differential expression and with a focus on cell biasing and fate splits at the level of bifurcations. It is meant to be fully integrated into scanpy ecosystem for seamless analysis of trajectories from single cell data of various modalities (e.g. RNA, ATAC).Competing Interest StatementThe authors have declared no competing interest.</p></div><div class=btn-links><a href=https://www.biorxiv.org/content/early/2022/07/10/2022.07.09.498657.full.pdf target=_blank rel=noopener><img class="btn btn-shield" src="https://img.shields.io/static/v1?logo=adobeacrobatreader&message=PDF&color=red&label=" alt=PDF></a>
<a href=https://github.com/LouisFaure/scFates target=_blank rel=noopener><img class="btn btn-shield" src="https://img.shields.io/static/v1?logo=github&message=Code&color=24292f&label=" alt=Code></a>
<button type=button class="btn js-cite-modal" style=padding:0 data-filename=/publication/faure2022/cite.bib>
<a><img class="btn btn-shield" src=https://img.shields.io/badge/BibTex-Cite-blueviolet alt=Cite></a></button>
<a href=https://doi.org/10.1101/2022.07.09.498657><img class="btn btn-shield" src="https://img.shields.io/static/v1?&message=10.1101%2f2022.07.09.498657&color=blue&label=DOI" alt=DOI></a>
<span class="__dimensions_badge_embed__ btn btn-shield" data-doi=10.1101/2022.07.09.498657 data-style=large_rectangle></span><script async src=https://badge.dimensions.ai/badge.js></script></div></div><div class=card-simple><div class=article-metadata><div><span>C. Petitpré</span>*, <span class=font-weight-bold><a href=/authors/admin/>L. Faure</a></span>*, <span>P. Uhl</span>, <span>P. Fontanet</span>, <span>I. Filova</span>, <span>G. Pavlinkova</span>, <span>I. Adameyko</span>, <span>S. Hadjab</span>, <span>F. Lallemend</span> (* co-first author)</div><span class=article-date>July 2022</span>
<span class=middot-divider></span><span class=pub-publication><em>Nature Communications</em></span></div><a href=/publication/petitpre2022/><img src=/publication/petitpre2022/featured_hud0c0331b4e3da3b6f5a3134418c74cf9_109578_918x517_fill_q90_lanczos_smart1.jpg class=article-banner alt></a><h3 class="article-title mb-1 mt-3"><a href=/publication/petitpre2022/>Single-cell RNA-sequencing analysis of the developing mouse inner ear identifies molecular logic of auditory neuron diversification</a></h3><div class=article-style><p>Different types of spiral ganglion neurons (SGNs) are essential for auditory perception by transmitting complex auditory information from hair cells (HCs) to the brain. Here, we use deep, single cell transcriptomics to study the molecular mechanisms that govern their identity and organization in mice. We identify a core set of temporally patterned genes and gene regulatory networks that may contribute to the diversification of SGNs through sequential binary decisions and demonstrate a role for NEUROD1 in driving specification of a Ic-SGN phenotype. We also find that each trajectory of the decision tree is defined by initial co-expression of alternative subtype molecular controls followed by gradual shifts toward cell fate resolution. Finally, analysis of both developing SGN and HC types reveals cell-cell signaling potentially playing a role in the differentiation of SGNs. Our results indicate that SGN identities are drafted prior to birth and reveal molecular principles that shape their differentiation and will facilitate studies of their development, physiology, and dysfunction. This study chronicles dynamic gene expression in differentiating spiral ganglion neurons from murine embryonic day 14.5 to postnatal stage, establishes their lineage trajectories, and identifies molecular determinants of cell fate decision.</p></div><div class=btn-links><a href=https://www.nature.com/articles/s41467-022-31580-1.pdf target=_blank rel=noopener><img class="btn btn-shield" src="https://img.shields.io/static/v1?logo=adobeacrobatreader&message=PDF&color=red&label=" alt=PDF></a>
<a href=https://github.com/LouisFaure/sgnfates_paper target=_blank rel=noopener><img class="btn btn-shield" src="https://img.shields.io/static/v1?logo=github&message=Code&color=24292f&label=" alt=Code></a>
<button type=button class="btn js-cite-modal" style=padding:0 data-filename=/publication/petitpre2022/cite.bib>
<a><img class="btn btn-shield" src=https://img.shields.io/badge/BibTex-Cite-blueviolet alt=Cite></a></button>
<a href=https://doi.org/10.1038/s41467-022-31580-1><img class="btn btn-shield" src="https://img.shields.io/static/v1?&message=10.1038%2fs41467-022-31580-1&color=blue&label=DOI" alt=DOI></a>
<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE165502" target=_blank rel=noopener><img class="btn btn-shield" src=https://img.shields.io/badge/Dataset-download-green alt=Dataset></a>
<span class="__dimensions_badge_embed__ btn btn-shield" data-doi=10.1038/s41467-022-31580-1 data-style=large_rectangle></span><script async src=https://badge.dimensions.ai/badge.js></script></div></div><div class=card-simple><div class=article-metadata><div><span>T. Bouderlique</span>*, <span>J. Petersen</span>*, <span class=font-weight-bold><a href=/authors/admin/>L. Faure</a></span>*, <span>D. Abed-Navandi</span>, <span>A. Bouchnita</span>, <span>B. Mueller</span>, <span>M. Nazarov</span>, <span>L. Englmaier</span>, <span>M. Tesarova</span>, <span>P. R. Frade</span>, <span>T. Zikmund</span>, <span>T. Koehne</span>, <span>J. Kaiser</span>, <span>K. Fried</span>, <span>C. Wild</span>, <span>O. Pantos</span>, <span>A. Hellander</span>, <span>J. Bythell</span>, <span>I. Adameyko</span> (* co-first author)</div><span class=article-date>May 2022</span>
<span class=middot-divider></span><span class=pub-publication><em>Current Biology</em></span></div><a href=/publication/bouderlique2022/><img src=/publication/bouderlique2022/featured_hu3fef9e3e959daa77096787a195c5f895_158749_918x517_fill_q90_lanczos_smart1.jpg class=article-banner alt></a><h3 class="article-title mb-1 mt-3"><a href=/publication/bouderlique2022/>Surface flow for colonial integration in reef-building corals</a></h3><div class=article-style><p>Reef-building corals are endangered animals with a complex colonial organization. Physiological mechanisms connecting multiple polyps and integrating them into a coral colony are still enigmatic. Using live imaging, particle tracking, and mathematical modeling, we reveal how corals connect individual polyps and form integrated polyp groups via species-specific, complex, and stable networks of currents at their surface. These currents involve surface mucus of different concentrations, which regulate joint feeding of the colony. Inside the coral, within the gastrovascular system, we expose the complexity of bidirectional branching streams that connect individual polyps. This system of canals extends the surface area by 4-fold and might improve communication, nutrient supply, and symbiont transfer. Thus, individual polyps integrate via complex liquid dynamics on the surface and inside the colony.</p></div><div class=btn-links><a href="https://www.cell.com/action/showPdf?pii=S0960-9822%2822%2900672-8" target=_blank rel=noopener><img class="btn btn-shield" src="https://img.shields.io/static/v1?logo=adobeacrobatreader&message=PDF&color=red&label=" alt=PDF></a>
<a href=https://github.com/LouisFaure/coral_paper/ target=_blank rel=noopener><img class="btn btn-shield" src="https://img.shields.io/static/v1?logo=github&message=Code&color=24292f&label=" alt=Code></a>
<button type=button class="btn js-cite-modal" style=padding:0 data-filename=/publication/bouderlique2022/cite.bib>
<a><img class="btn btn-shield" src=https://img.shields.io/badge/BibTex-Cite-blueviolet alt=Cite></a></button>
<a href=https://doi.org/10.1016/j.cub.2022.04.054><img class="btn btn-shield" src="https://img.shields.io/static/v1?&message=10.1016%2fj.cub.2022.04.054&color=blue&label=DOI" alt=DOI></a>
<span class="__dimensions_badge_embed__ btn btn-shield" data-doi=10.1016/j.cub.2022.04.054 data-style=large_rectangle></span><script async src=https://badge.dimensions.ai/badge.js></script></div></div><div class=card-simple><div class=article-metadata><div><span class=font-weight-bold><a href=/authors/admin/>L. Faure</a></span>, <span>Y. Wang</span>, <span>M. Eleni Kastriti</span>, <span>P. Fontanet</span>, <span>K. K Y Cheung</span>, <span>C. Petitpré</span>, <span>H. Wu</span>, <span>L. Linyu Sun</span>, <span>K. Runge</span>, <span>L. Croci</span>, <span>M. A Landy</span>, <span>H. C Lai</span>, <span>G. Giacomo Consalez</span>, <span>A. De Chevigny</span>, <span>F. Lallemend</span>, <span>I. Adameyko</span>, <span>S. Hadjab</span></div><span class=article-date>January 2020</span>
<span class=middot-divider></span><span class=pub-publication><em>Nature Communications</em></span></div><a href=/publication/faure2020/><img src=/publication/faure2020/featured_hu8a88503e6d3c944c0a74a69373f32ce7_283386_918x517_fill_q90_lanczos_smart1.jpg class=article-banner alt></a><h3 class="article-title mb-1 mt-3"><a href=/publication/faure2020/>Single cell RNA sequencing identifies early diversity of sensory neurons forming via bi-potential intermediates</a></h3><div class=article-style><p>Somatic sensation is defined by the existence of a diversity of primary sensory neurons with unique biological features and response profiles to external and internal stimuli. However, there is no coherent picture about how this diversity of cell states is transcriptionally generated. Here, we use deep single cell analysis to resolve fate splits and molecular biasing processes during sensory neurogenesis in mice. Our results identify a complex series of successive and specific transcriptional changes in post-mitotic neurons that delineate hierarchical regulatory states leading to the generation of the main sensory neuron classes. In addition, our analysis identifies previously undetected early gene modules expressed long before fate determination although being clearly associated with defined sensory subtypes. Overall, the early diversity of sensory neurons is generated through successive bi-potential intermediates in which synchronization of relevant gene modules and concurrent repression of competing fate programs precede cell fate stabilization and final commitment.</p></div><div class=btn-links><a href=https://www.nature.com/articles/s41467-020-17929-4.pdf target=_blank rel=noopener><img class="btn btn-shield" src="https://img.shields.io/static/v1?logo=adobeacrobatreader&message=PDF&color=red&label=" alt=PDF></a>
<a href=https://github.com/LouisFaure/sensoryfates_paper target=_blank rel=noopener><img class="btn btn-shield" src="https://img.shields.io/static/v1?logo=github&message=Code&color=24292f&label=" alt=Code></a>
<button type=button class="btn js-cite-modal" style=padding:0 data-filename=/publication/faure2020/cite.bib>
<a><img class="btn btn-shield" src=https://img.shields.io/badge/BibTex-Cite-blueviolet alt=Cite></a></button>
<a href=https://doi.org/10.1038/s41467-020-17929-4><img class="btn btn-shield" src="https://img.shields.io/static/v1?&message=10.1038%2fs41467-020-17929-4&color=blue&label=DOI" alt=DOI></a>
<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE150150" target=_blank rel=noopener><img class="btn btn-shield" src=https://img.shields.io/badge/Dataset-download-green alt=Dataset></a>
<span class="__dimensions_badge_embed__ btn btn-shield" data-doi=10.1038/s41467-020-17929-4 data-style=large_rectangle></span><script async src=https://badge.dimensions.ai/badge.js></script></div></div></div></div></div></section><section id=publications class="home-section wg-pages" style="padding:20px 0"><div class=container><div class=row><div class="col-12 col-lg-4 section-heading"><h1>Recent Publications</h1></div><div class="col-12 col-lg-8"><div class="media stream-item"><div class=media-body><h3 class="article-title mb-0 mt-0"><a href=/publication/faure2022/>scFates: a scalable python package for advanced pseudotime and bifurcation analysis from single cell data</a></h3><h2 class=article-style><em>bioRxiv</em>,
(2022)</h2><div class="stream-meta article-metadata"><div><span class=font-weight-bold><a href=/authors/admin/>L. Faure</a></span>, <span>R. Soldatov</span>, <span>P. V. Kharchenko</span>, <span>I. Adameyko</span></div></div><div class=btn-links><a href=https://www.biorxiv.org/content/early/2022/07/10/2022.07.09.498657.full.pdf target=_blank rel=noopener><img class="btn btn-shield" src="https://img.shields.io/static/v1?logo=adobeacrobatreader&message=PDF&color=red&label=" alt=PDF></a>
<a href=https://github.com/LouisFaure/scFates target=_blank rel=noopener><img class="btn btn-shield" src="https://img.shields.io/static/v1?logo=github&message=Code&color=24292f&label=" alt=Code></a>
<button type=button class="btn js-cite-modal" style=padding:0 data-filename=/publication/faure2022/cite.bib>
<a><img class="btn btn-shield" src=https://img.shields.io/badge/BibTex-Cite-blueviolet alt=Cite></a></button>
<a href=https://doi.org/10.1101/2022.07.09.498657><img class="btn btn-shield" src="https://img.shields.io/static/v1?&message=10.1101%2f2022.07.09.498657&color=blue&label=DOI" alt=DOI></a>
<span class="__dimensions_badge_embed__ btn btn-shield" data-doi=10.1101/2022.07.09.498657 data-style=large_rectangle></span><script async src=https://badge.dimensions.ai/badge.js></script></div></div><div class=ml-3><a href=/publication/faure2022/><img src=/publication/faure2022/featured_hu6a7e5c640ce1396b72b3025c016be542_70624_150x0_resize_q90_lanczos.jpg alt></a></div></div><div class="media stream-item"><div class=media-body><h3 class="article-title mb-0 mt-0"><a href=/publication/petitpre2022/>Single-cell RNA-sequencing analysis of the developing mouse inner ear identifies molecular logic of auditory neuron diversification</a></h3><h2 class=article-style><em>Nature Communications</em>,
(2022)</h2><div class="stream-meta article-metadata"><div><span>C. Petitpré</span>*, <span class=font-weight-bold><a href=/authors/admin/>L. Faure</a></span>*, <span>P. Uhl</span>, <span>P. Fontanet</span>, <span>I. Filova</span>, <span>G. Pavlinkova</span>, <span>I. Adameyko</span>, <span>S. Hadjab</span>, <span>F. Lallemend</span> (* co-first author)</div></div><div class=btn-links><a href=https://www.nature.com/articles/s41467-022-31580-1.pdf target=_blank rel=noopener><img class="btn btn-shield" src="https://img.shields.io/static/v1?logo=adobeacrobatreader&message=PDF&color=red&label=" alt=PDF></a>
<a href=https://github.com/LouisFaure/sgnfates_paper target=_blank rel=noopener><img class="btn btn-shield" src="https://img.shields.io/static/v1?logo=github&message=Code&color=24292f&label=" alt=Code></a>
<button type=button class="btn js-cite-modal" style=padding:0 data-filename=/publication/petitpre2022/cite.bib>
<a><img class="btn btn-shield" src=https://img.shields.io/badge/BibTex-Cite-blueviolet alt=Cite></a></button>
<a href=https://doi.org/10.1038/s41467-022-31580-1><img class="btn btn-shield" src="https://img.shields.io/static/v1?&message=10.1038%2fs41467-022-31580-1&color=blue&label=DOI" alt=DOI></a>
<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE165502" target=_blank rel=noopener><img class="btn btn-shield" src=https://img.shields.io/badge/Dataset-download-green alt=Dataset></a>
<span class="__dimensions_badge_embed__ btn btn-shield" data-doi=10.1038/s41467-022-31580-1 data-style=large_rectangle></span><script async src=https://badge.dimensions.ai/badge.js></script></div></div><div class=ml-3><a href=/publication/petitpre2022/><img src=/publication/petitpre2022/featured_hud0c0331b4e3da3b6f5a3134418c74cf9_109578_150x0_resize_q90_lanczos.jpg alt></a></div></div><div class="media stream-item"><div class=media-body><h3 class="article-title mb-0 mt-0"><a href=/publication/akkuratova2022/>Developmental heterogeneity of embryonic neuroendocrine chromaffin cells and their maturation dynamics</a></h3><h2 class=article-style><em>bioRxiv</em>,
(2022)</h2><div class="stream-meta article-metadata"><div><span>N. Akkuratova</span>*, <span class=font-weight-bold><a href=/authors/admin/>L. Faure</a></span>*, <span>P. Kameneva</span>, <span>M. Eleni Kastriti</span>, <span>I. Adameyko</span> (* co-first author)</div></div><div class=btn-links><a href=https://www.biorxiv.org/content/early/2022/05/26/2022.05.26.493613.full.pdf target=_blank rel=noopener><img class="btn btn-shield" src="https://img.shields.io/static/v1?logo=adobeacrobatreader&message=PDF&color=red&label=" alt=PDF></a>
<a href=https://github.com/LouisFaure/ChromaffinFates_paper target=_blank rel=noopener><img class="btn btn-shield" src="https://img.shields.io/static/v1?logo=github&message=Code&color=24292f&label=" alt=Code></a>
<button type=button class="btn js-cite-modal" style=padding:0 data-filename=/publication/akkuratova2022/cite.bib>
<a><img class="btn btn-shield" src=https://img.shields.io/badge/BibTex-Cite-blueviolet alt=Cite></a></button>
<a href=https://doi.org/10.1101/2022.05.26.493613><img class="btn btn-shield" src="https://img.shields.io/static/v1?&message=10.1101%2f2022.05.26.493613&color=blue&label=DOI" alt=DOI></a>
<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE204700" target=_blank rel=noopener><img class="btn btn-shield" src=https://img.shields.io/badge/Dataset-download-green alt=Dataset></a>
<span class="__dimensions_badge_embed__ btn btn-shield" data-doi=10.1101/2022.05.26.493613 data-style=large_rectangle></span><script async src=https://badge.dimensions.ai/badge.js></script></div></div><div class=ml-3></div></div><div class="media stream-item"><div class=media-body><h3 class="article-title mb-0 mt-0"><a href=/publication/kameneva2022/>Serotonin limits generation of chromaffin cells during adrenal organ development</a></h3><h2 class=article-style><em>Nature Communications</em>,
(2022)</h2><div class="stream-meta article-metadata"><div><span>P. Kameneva</span>*, <span>V. I. Melnikova</span>*, <span>M. Eleni Kastriti</span>, <span>A. Kurtova</span>, <span>E. Kryukov</span>, <span>A. Murtazina</span>, <span class=font-weight-bold><a href=/authors/admin/>L. Faure</a></span>, <span>I. Poverennaya</span>, <span>A. V. Artemov</span>, <span>T. S. Kalinina</span>, <span>N. V. Kudryashov</span>, <span>M. Bader</span>, <span>J. Skoda</span>, <span>P. Chlapek</span>, <span>L. Curylova</span>, <span>L. Sourada</span>, <span>J. Neradil</span>, <span>M. Tesarova</span>, <span>M. Pasqualetti</span>, <span>P. Gaspar</span>, <span>V. D. Yakushov</span>, <span>B. I. Sheftel</span>, <span>T. Zikmund</span>, <span>J. Kaiser</span>, <span>K. Fried</span>, <span>N. Alenina</span>, <span>E. E. Voronezhskaya</span>, <span>I. Adameyko</span> (* co-first author)</div></div><div class=btn-links><a href=https://www.nature.com/articles/s41467-022-30438-w.pdf target=_blank rel=noopener><img class="btn btn-shield" src="https://img.shields.io/static/v1?logo=adobeacrobatreader&message=PDF&color=red&label=" alt=PDF></a>
<button type=button class="btn js-cite-modal" style=padding:0 data-filename=/publication/kameneva2022/cite.bib>
<a><img class="btn btn-shield" src=https://img.shields.io/badge/BibTex-Cite-blueviolet alt=Cite></a></button>
<a href=https://doi.org/10.1038/s41467-022-30438-w><img class="btn btn-shield" src="https://img.shields.io/static/v1?&message=10.1038%2fs41467-022-30438-w&color=blue&label=DOI" alt=DOI></a>
<span class="__dimensions_badge_embed__ btn btn-shield" data-doi=10.1038/s41467-022-30438-w data-style=large_rectangle></span><script async src=https://badge.dimensions.ai/badge.js></script></div></div><div class=ml-3></div></div><div class="media stream-item"><div class=media-body><h3 class="article-title mb-0 mt-0"><a href=/publication/bouderlique2022/>Surface flow for colonial integration in reef-building corals</a></h3><h2 class=article-style><em>Current Biology</em>,
(2022)</h2><div class="stream-meta article-metadata"><div><span>T. Bouderlique</span>*, <span>J. Petersen</span>*, <span class=font-weight-bold><a href=/authors/admin/>L. Faure</a></span>*, <span>D. Abed-Navandi</span>, <span>A. Bouchnita</span>, <span>B. Mueller</span>, <span>M. Nazarov</span>, <span>L. Englmaier</span>, <span>M. Tesarova</span>, <span>P. R. Frade</span>, <span>T. Zikmund</span>, <span>T. Koehne</span>, <span>J. Kaiser</span>, <span>K. Fried</span>, <span>C. Wild</span>, <span>O. Pantos</span>, <span>A. Hellander</span>, <span>J. Bythell</span>, <span>I. Adameyko</span> (* co-first author)</div></div><div class=btn-links><a href="https://www.cell.com/action/showPdf?pii=S0960-9822%2822%2900672-8" target=_blank rel=noopener><img class="btn btn-shield" src="https://img.shields.io/static/v1?logo=adobeacrobatreader&message=PDF&color=red&label=" alt=PDF></a>
<a href=https://github.com/LouisFaure/coral_paper/ target=_blank rel=noopener><img class="btn btn-shield" src="https://img.shields.io/static/v1?logo=github&message=Code&color=24292f&label=" alt=Code></a>
<button type=button class="btn js-cite-modal" style=padding:0 data-filename=/publication/bouderlique2022/cite.bib>
<a><img class="btn btn-shield" src=https://img.shields.io/badge/BibTex-Cite-blueviolet alt=Cite></a></button>
<a href=https://doi.org/10.1016/j.cub.2022.04.054><img class="btn btn-shield" src="https://img.shields.io/static/v1?&message=10.1016%2fj.cub.2022.04.054&color=blue&label=DOI" alt=DOI></a>
<span class="__dimensions_badge_embed__ btn btn-shield" data-doi=10.1016/j.cub.2022.04.054 data-style=large_rectangle></span><script async src=https://badge.dimensions.ai/badge.js></script></div></div><div class=ml-3><a href=/publication/bouderlique2022/><img src=/publication/bouderlique2022/featured_hu3fef9e3e959daa77096787a195c5f895_158749_150x0_resize_q90_lanczos.jpg alt></a></div></div><div class=see-all><a href=/publication/>See all publications
<i class="fas fa-angle-right"></i></a></div></div></div></div></section><section id=experience class="home-section wg-experience" style="padding:20px 0"><div class=container><div class=row><div class="col-12 col-lg-4 section-heading"><h1>Experience</h1></div><div class="col-12 col-lg-8"><div class="row experience"><div class="col-auto text-center flex-column d-none d-sm-flex"><div class="row h-50"><div class=col>&nbsp;</div><div class=col>&nbsp;</div></div><div class=m-2><span class="badge badge-pill border">&nbsp;</span></div><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div></div><div class="col py-2"><div class=card><div class=card-body><div class="d-flex align-content-start"><div class="mr-2 mb-2"><img src=/media/icons/brands/EMBL_EBI.svg width=56px height=56px alt="EMBL-EBI: NEUcrest Big Data course" loading=lazy></div><div><div class="section-subheading card-title exp-title text-muted my-0">Lecturer</div><div class="section-subheading card-title exp-company text-muted my-0">EMBL-EBI: NEUcrest Big Data course</div><div class="text-muted exp-meta">April 2021 –
April 2021
<span class=middot-divider></span><span>Online</span></div></div></div><div class=card-text>Presentation about methods related to trajectory inference analysis.</div></div></div></div></div><div class="row experience"><div class="col-auto text-center flex-column d-none d-sm-flex"><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div><div class=m-2><span class="badge badge-pill border">&nbsp;</span></div><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div></div><div class="col py-2"><div class=card><div class=card-body><div class="d-flex align-content-start"><div class="mr-2 mb-2"><img src=/media/icons/brands/Inserm.svg width=56px height=56px alt="INSERM U900 Bioinformatics and computational biology of cancer unit" loading=lazy></div><div><div class="section-subheading card-title exp-title text-muted my-0">Intern</div><div class="section-subheading card-title exp-company text-muted my-0">INSERM U900 Bioinformatics and computational biology of cancer unit</div><div class="text-muted exp-meta">January 2018 –
June 2018
<span class=middot-divider></span><span>Curie Institute, Paris</span></div></div></div><div class=card-text>Gene Regulation Network inference using pseudotime ordering of scRNAseq data.</div></div></div></div></div><div class="row experience"><div class="col-auto text-center flex-column d-none d-sm-flex"><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div><div class=m-2><span class="badge badge-pill border">&nbsp;</span></div><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div></div><div class="col py-2"><div class=card><div class=card-body><div class="d-flex align-content-start"><div class="mr-2 mb-2"><img src=/media/icons/brands/Huberlin.svg width=56px height=56px alt="Institute for Theoretical Biology" loading=lazy></div><div><div class="section-subheading card-title exp-title text-muted my-0">Intern</div><div class="section-subheading card-title exp-company text-muted my-0">Institute for Theoretical Biology</div><div class="text-muted exp-meta">April 2017 –
July 2017
<span class=middot-divider></span><span>Humboldt University, Berlin</span></div></div></div><div class=card-text>Mathematical modeling of microbial metabolism &amp; cyanobacterial phototrophic growth. Kinetic modeling and flux balance analysis.</div></div></div></div></div><div class="row experience"><div class="col-auto text-center flex-column d-none d-sm-flex"><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div><div class=m-2><span class="badge badge-pill border">&nbsp;</span></div><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div></div><div class="col py-2"><div class=card><div class=card-body><div class="d-flex align-content-start"><div class="mr-2 mb-2"><img src=/media/icons/brands/Inserm.svg width=56px height=56px alt="Inserm u1001 Research Lab" loading=lazy></div><div><div class="section-subheading card-title exp-title text-muted my-0">Intern</div><div class="section-subheading card-title exp-company text-muted my-0">Inserm u1001 Research Lab</div><div class="text-muted exp-meta">January 2017 –
April 2017
<span class=middot-divider></span><span>Descartes University, Paris</span></div></div></div><div class=card-text>Heterogeneity of the spontaneous DNA replication errors in single isogenic <em>E.coli</em> cells.</div></div></div></div></div><div class="row experience"><div class="col-auto text-center flex-column d-none d-sm-flex"><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div><div class=m-2><span class="badge badge-pill border">&nbsp;</span></div><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div></div><div class="col py-2"><div class=card><div class=card-body><div class="d-flex align-content-start"><div class="mr-2 mb-2"><img src=/media/icons/brands/ESPCI.svg width=56px height=56px alt="Laboratory of Biochemistery" loading=lazy></div><div><div class="section-subheading card-title exp-title text-muted my-0">Intern</div><div class="section-subheading card-title exp-company text-muted my-0">Laboratory of Biochemistery</div><div class="text-muted exp-meta">September 2016 –
December 2016
<span class=middot-divider></span><span>ESPCI, Paris</span></div></div></div><div class=card-text>Abzymes Project: Quantitative selection and evolution at by high throughput screening of antibodies libraries for DNAse reaction, using microfluidics.</div></div></div></div></div><div class="row experience"><div class="col-auto text-center flex-column d-none d-sm-flex"><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div><div class=m-2><span class="badge badge-pill border">&nbsp;</span></div><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div></div><div class="col py-2"><div class=card><div class=card-body><div class="d-flex align-content-start"><div class="mr-2 mb-2"><img src=/media/icons/brands/Pasteur.svg width=56px height=56px alt="Microbial Morphogenesis and Growth Lab" loading=lazy></div><div><div class="section-subheading card-title exp-title text-muted my-0">Intern</div><div class="section-subheading card-title exp-company text-muted my-0">Microbial Morphogenesis and Growth Lab</div><div class="text-muted exp-meta">January 2016 –
July 2016
<span class=middot-divider></span><span>Pasteur Institute, Paris</span></div></div></div><div class=card-text>Single-cell study, tracking of the starting of chromosomal segregation over several generations in <em>E. coli</em>, using fluorescence microscopy and quantitative image analysis.</div></div></div></div></div><div class="row experience"><div class="col-auto text-center flex-column d-none d-sm-flex"><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div><div class=m-2><span class="badge badge-pill border">&nbsp;</span></div><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div></div><div class="col py-2"><div class=card><div class=card-body><div class="d-flex align-content-start"><div class="mr-2 mb-2"><img src=/media/icons/brands/Petnica.svg width=56px height=56px alt="Labsprint at Petnica Science Center" loading=lazy></div><div><div class="section-subheading card-title exp-title text-muted my-0">Teaching Assistant</div><div class="section-subheading card-title exp-company text-muted my-0">Labsprint at Petnica Science Center</div><div class="text-muted exp-meta">August 2016 –
August 2016
<span class=middot-divider></span><span>Petnica, Serbia</span></div></div></div><div class=card-text>Worked as a teaching assistant for 2nd year students from Undergraduate program “Frontiers in Life Science”. Helping the lecturer for the setup of the courses, preparing lab supplies, Giving advices to the students for their projects.</div></div></div></div></div><div class="row experience"><div class="col-auto text-center flex-column d-none d-sm-flex"><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div><div class=m-2><span class="badge badge-pill border">&nbsp;</span></div><div class="row h-50"><div class=col>&nbsp;</div><div class=col>&nbsp;</div></div></div><div class="col py-2"><div class=card><div class=card-body><div class="d-flex align-content-start"><div class="mr-2 mb-2"><img src=/media/icons/brands/Curie.svg width=56px height=56px alt="Laboratory of Molecular and Cellular Biology of Eukaryotes" loading=lazy></div><div><div class="section-subheading card-title exp-title text-muted my-0">Intern</div><div class="section-subheading card-title exp-company text-muted my-0">Laboratory of Molecular and Cellular Biology of Eukaryotes</div><div class="text-muted exp-meta">January 2015 –
June 2015
<span class=middot-divider></span><span>Curie Institute, Paris</span></div></div></div><div class=card-text>Study of regulations of mitochondrial fusion/fission mechanisms in <em>S. cerevisiae</em>. Gene tagging/deletion; Glycerol growth analysis; Microscopy with GFP-tagged strains; Cell transformation.</div></div></div></div></div></div></div></div></section><section id=contact class="home-section wg-contact" style="padding:20px 0 200px"><div class=container><div class="row contact-widget"><div class="col-12 col-lg-4 section-heading"><h1>Contact</h1></div><div class="col-12 col-lg-8"><ul class=fa-ul><li><i class="fa-li fas fa-envelope fa-2x" aria-hidden=true></i><span id=person-email><a href=mailto:louis.faure@meduniwien.ac.at>louis.faure@meduniwien.ac.at</a></span></li><li><i class="fa-li fas fa-map-marker fa-2x" aria-hidden=true></i><span id=person-address>4 Spitalgasse, Wien, 1190, Austria</span></li></ul><div class=d-none><input id=map-provider value=2>
<input id=map-lat value=48.218520>
<input id=map-lng value=16.352059>
<input id=map-dir value="4 Spitalgasse, Wien, 1190, Austria">
<input id=map-zoom value=15>
<input id=map-api-key value></div><div id=map></div></div></div></div></section><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js integrity="sha256-yt2kYMy0w8AbtF89WXb2P1rfjcP/HTHLT7097U8Y5b8=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.10/highlight.min.js integrity="sha256-1zu+3BnLYV9LdiY85uXMzii3bdrkelyp37e0ZyTAQh0=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.10/languages/r.min.js></script><script src=https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.5.1/leaflet.js integrity="sha256-EErZamuLefUnbMBQbsEqu1USa+btR2oIlCpBJbyD4/g=" crossorigin=anonymous></script><script>hljs.initHighlightingOnLoad();</script><script>const search_config={"indexURI":"/index.json","minLength":1,"threshold":0.3};const i18n={"no_results":"No results found","placeholder":"Search...","results":"results found"};const content_type={'post':"Posts",'project':"Projects",'publication':"Publications",'talk':"Talks"};</script><script id=search-hit-fuse-template type=text/x-template>
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script><script src=https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin=anonymous></script><script src=/js/academic.min.d6bd04fdad2ad213aa8111c5a3b72fc5.js></script><div class=container><footer class=site-footer><p class=powered-by>Powered by the
<a href=https://sourcethemes.com/academic/ target=_blank rel=noopener>Academic theme</a> for
<a href=https://gohugo.io target=_blank rel=noopener>Hugo</a>.
<span class=float-right aria-hidden=true><a href=# class=back-to-top><span class=button_icon><i class="fas fa-chevron-up fa-2x"></i></span></a></span></p></footer></div><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&times;</span></button></div><div class=modal-body><pre><code class="tex hljs"></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i>Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i>Download</a><div id=modal-error></div></div></div></div></div></body></html>